packages/components/tests/integration/components/hds/tooltip/tooltip-helper-test.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { focus, render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Component | hds/tooltip/index', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it attaches a tooltip to the element it is invoked on', async function (assert) {
+    await render(
+      hbs`<a href="#" {{hds-tooltip "More info."}} data-test-tooltip-helper>Info</a>`
+    );
+    // activate the tooltip:
+    await focus('[data-test-tooltip-helper]');
+    // test that the tooltip exists and has the passed in content:
+    assert.dom('.tippy-content').hasText('More info.');
+
+    // test the expected accessibility related attributes:
+    assert.dom('[data-test-tooltip-helper]').hasAttribute('aria-describedby');
+    assert.dom('[data-tippy-root]').hasAttribute('id');
+    assert.strictEqual(
+      this.element
+        .querySelector('[data-test-tooltip-helper]')
+        .getAttribute('aria-describedby'),
+      this.element.querySelector('[data-tippy-root]').getAttribute('id')
+    );
+  });
+});
```

Comment:
Most of the tooltip features are tested on the component version so I didn't think I needed to be too exhaustive for the helper tests but let me know if there are more tests I should add.